README.asciidoc

```diff
@@ -304,22 +304,52 @@ Please let us know if you have any ideas about how to make the default config mo
 ==== Server-specific configuration
 
 Many servers accept configuration options that are not part of the LSP spec.  The TOML table
-`[language.<filetype>.settings]` holds those configuration options.  It has the same structure
+`[language_server.<server_name>.settings]` holds those configuration options.  It has the same structure
```

Comment:
I believe that if the old format (`[language.c_cpp]`) is used, then we should keep sending `c_cpp` as languageId, not the filetype. This is to avoid breaking existing configs. Unless we find that all filetypes work fine but I doubt that (for example with typescript-language-server, languageId javascript didn't work for jsx files last time I checked).
To implement this, we probabl need a custom deserializer.